setup.py

```diff
@@ -27,20 +27,20 @@
 history = open('HISTORY.rst').read().replace('.. :changelog:', '')
 
 setup(
-    name='django-oidc',
+    name='django-oidc-tf',
```

Comment:
We will want to keep the original name

Reply:
Thanks for the review! Some changes are come from the fork py-pa/django-oidc which I merged to get both py3 and py2 support (I use this library both for 2 and 3), so can't say why it's been done what way - like this one.
But I'll try to handle issues which may be handled at all. Looks like some potential bugs were introduced so my projects are affected so it will be fixed.